public/index.html

```diff
@@ -151,7 +151,12 @@
     <header>
       <h1 class="visually-hidden">Excalidraw</h1>
     </header>
-    <div id="root"></div>
+
+    <!-- Include padding for container delta check -->
```

Comment:
I'm not happy with this change. We should find another solution that doesn't affect the production version.

Reply:
OK, i will revert this.